### ISSUE TITLE
[FEATURE REQUEST] A Way To Discover New Shows On Rlsbb

### ISSUE THREAD

**Issue by mo123** [LivingWithHippos/unchained-android#348]
**Describe the solution you'd like**
A way to browse, discover the latest free-to-air tv show suggestions posted on rlsbb and resolve the hoster links to real-debrid without first knowing what to search for.
There are lots of free-to-air reality, day-time shows(non tv-series) for US citizens not available to buy or found on torrents but indexed on rlsbb.

Can make use of rss feeds
https://log.rlsbb.ru/category/tv-shows/feed/ or
rlsbb's log site to discover new shows.
https://log.rlsbb.ru/category/tv-shows

**Describe alternatives you've considered**
A stremio addon for rlsbb show discovery, unchained would be ideal.

**Comment by Invictaz** [comment]
So you want an internal RSS viewer to import the urls?

**Comment by mo123** [comment]
Yes or some way to see the most recent tv show/movie links posted in the last 24 hours.
It should then only show the rapidgator, nitroshare etc. links and skip the rest.
After clicking on a link it should send it to real-debrid so it can be streamed/downloaded later.
